src/BucketTesting/Domain/Model/BucketLog.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\BucketTesting\Domain\Model;
+
+use DateTime;
+use DateTimeInterface;
+
+class BucketLog {
+
+	private int $id;
+	private int $external_id;
+	private string $eventName;
+	private array $buckets;
+	private DateTimeInterface $date;
+
+	public function __construct( int $external_id, string $eventName, array $buckets, ?DateTime $date = null ) {
```

Comment:
If we didn't have the optional date param, a better way to type the buckets would be with the spread operator. But in this case, please provide a docblock with type information for `$buckets`.